Removed "Core Analytics" check from Akamai

wxwr opened the pull request:
Does Akamai support "Core Analytics"?
https://docs.microsoft.com/en-us/azure/cdn/cdn-analyze-usage-patterns
This feature is available with Azure CDN from Verizon products (Standard and Premium). **It is not supported on Azure CDN from Akamai.** For a comparison of CDN features, see Azure CDN Overview.

azurecla commented:
Hi __@astaykov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Removed "Core Analytics" check from Akamai

Does Akamai support Core analytics?
https://docs.microsoft.com/en-us/azure/cdn/cdn-analyze-usage-patterns
This feature is available with Azure CDN from Verizon products (Standard and Premium). It is not supported on Azure CDN from Akamai. For a comparison of CDN features, see Azure CDN Overview.
